### ISSUE TITLE
Mask poorly aligned bases at beginning or end of input alignment!

### ISSUE THREAD

**Issue by seanmcallister** [mooreryan/ZetaHunter#31]
Have ZH remove lower case bases from SINA aligned input at the beginnings and ends of sequences so that bases that poorly align to the SILVA database are removed. These are likely adaptors/vector and should be removed anyway.

¡Important!

**Comment by mooreryan** [comment]
Should these be permanently removed at the beginning?

**Comment by seanmcallister** [comment]
Yes, I think so. **Both** _beginning_ and _end_ of the alignments. I wouldn't worry too much about the lowercase in the middle. Although, if there are a lot, I suppose that the sequence should be flagged. (although it will likely be called as a chimera).
